mappings/net/minecraft/command/argument/ItemSlotArgumentType.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/unmapped/C_jqbfufjx net/minecraft/command/argument/ItemSlotArgumentType
 	FIELD f_bsfrpwff UNKNOWN_SLOT_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD f_kfpmusct EXAMPLES Ljava/util/Collection;
+	FIELD f_opwacyge SINGLE_SLOT_ONLY_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

Comment:
```suggestion
	FIELD f_opwacyge ONLY_SINGLE_ALLOWED Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```
I don't think an `EXCEPTION` suffix is necessary here, but it does have a suffix it should be `EXCEPTION_TYPE`